radar-upload-backend/src/main/java/org/radarbase/upload/Config.kt

```diff
@@ -42,4 +42,6 @@ data class Config(
         var sourceTypes: List<SourceTypeDTO>? = null,
         var additionalPersistenceConfig: Map<String, String>? = null,
         var contentStreamBufferSize: Long = 1048576L,
-        var enableCors: Boolean? = false)
+        var enableCors: Boolean? = false,
+        var syncProjects: Long? = 30,
```

Comment:
rename to `syncProjectsIntervalMin`? Similarly for `syncParticipants`. I would suggest to make this a non-null field, so either this default is used or any user-provided value. This eliminates non-null coercion (`!!`) when evaluating the variables.